pkg/apis/core/types.go

```diff
@@ -2188,9 +2188,29 @@ type Container struct {
 	TTY bool
 }
 
-// Handler defines a specific action that should be taken
+// ProbeHandler defines a specific action that should be taken in a probe.
+// This type has a strong relationship to LifecycleHandler - overlapping fields
```

Comment:
why the comment? Is it a requirement to keep them identical in the long run? Even implementation is different (see https://github.com/kubernetes/kubernetes/pull/86139/).

Reply:
followed up with https://github.com/kubernetes/kubernetes/pull/106285